tests/IBTK/mpi_type_wrappers.output

```diff
@@ -0,0 +1,25 @@
+
+size of MPI type = 8
+sizeof(double) = 8
+
+size of MPI type = 4
+sizeof(int) = 4
+
+size of MPI type = 4
+sizeof(float) = 4
+
+size of MPI type = 1
+sizeof(char) = 1
+
+size of MPI type = 8
+sizeof(std::pair<float, int>) = 8
+sizeof(struct {float a; int b;}) = 8
+
+size of MPI type = 12
```

Comment:
The default behavior for GCC is to keep things aligned on 8-byte boundaries, but MPI will not use any padding to decrease message sizes. Hence these are not equal.